.gitignore

```diff
@@ -10,3 +10,4 @@ build
 docs/_build/
 .cache/
 .eggs/
+.*env*/
```

Comment:
Why did you add this?

Reply:
I'm sorry for adding this change to this pull request. It's for ignoring virtual environments.